core/templates/dev/head/pages/exploration_player/LearnerViewInfo.js

```diff
@@ -35,7 +35,11 @@ oppia.controller('LearnerViewInfo', [
           }
         }).then(function(response) {
           expInfo = response.data.summaries[0];
-          openInformationCardModal();
+          if (expInfo) {
+            openInformationCardModal();
+          } else {
+            $log.info('This Exploration is not yet published yet.');
```

Comment:
Not sure why we're logging stuff here, especially if the behaviour is expected. We should be handling the error case gracefully (see top-level comment).